pkg/event/event.go

```diff
@@ -0,0 +1,201 @@
+package event
```

Comment:
Is this just related to GNSS or all events?

Reply:
This will be for all events , this is the place you construct log string and write to socket for event framework to pick it up and process